docs/code.rst

```diff
@@ -507,6 +510,7 @@ Community contributed hooks
 .. autoclass:: airflow.contrib.hooks.gcp_function_hook.GcfHook
 .. autoclass:: airflow.contrib.hooks.gcp_api_base_hook.GoogleCloudBaseHook
 .. autoclass:: airflow.contrib.hooks.gcp_kms_hook.GoogleCloudKMSHook
+.. autoclass:: airflow.contrib.hooks.gcp_video_intelligence_hook.CloudVideoIntelligenceHook
```

Comment:
Please keep alphabetical order. You should sort first by classname. In next step, by full class path.

Reply:
Done

Reply:
Done